test/common/modules/shared_helper.js

```diff
@@ -44,6 +44,20 @@ define([
 
   async function monitorConnectionAsync(action, realtime, states) {
     const monitoringResultPromise = new Promise((resolve, reject) => {
+      if (Object.prototype.toString.call(realtime) == '[object Array]') {
```

Comment:
May be `Array.isArray(realtime)` here ?

Reply:
Yes probably better, I was following the pattern used in `closeAndFinish`